setup.cfg

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=test_settings
-addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8 --cov --nomigrations
+addopts = --verbose --nomigrations --reuse-db -s --ignore ve --flake8
```

Comment:
It's useful to have coverage on local runs without having to explicitly ask for it, but I don't think there's a sane way to do that and also skip coverage on pypy builds. :-/